ansible/templates/zapd_block_height.sh

```diff
@@ -1,15 +1,32 @@
 #!/bin/bash
 
-dest_email=alerts@zap.me
-remote_block=`curl -s https://testnode1.wavesnodes.com/blocks/height | jq '.["height"]'` 
-local_block=`curl -s localhost:6869/blocks/height | jq '.["height"]'`
+dest_email={{ dest_email }}
 max_height_diff=2
 
-### Remote block would be same/higher than local block.
-num=$(( $remote_block - $local_block ))
+{% if ansible_hostname == 'testnet' %}
```

Comment:
see below.. use the TESTNET variable already defined project wide by ansible_deploy.sh